doc/Start.md

```diff
@@ -1,53 +1,251 @@
+
+You need a game server that is compatible with the original game. There are many implementations/versions/forks that are compatible, you can use any of them, but we suggest using one of the following two, because we test using these emulators:
+  
+- [rAthena](https://github.com/rathena/rathena)
+- [Hercules](https://github.com/HerculesWS/Hercules/)
+
+Useful information: 
 - Disable pincode on the game server. (Not supported yet)
 - Disable packet_obfuscation on the game server. (Not supported yet, causes invalid packets)
-#### Browser
-To run roBrowser you will need an up to date browser that supports [WebGL](http://www.chromeexperiments.com/webgl/) and is OpenGL ES 2.0 compatible. We've tested the following browsers:
-* Chrome _(Desktop & Mobile)_
-* FireFox _(Desktop & Mobile)_
-* Safari _(Desktop & Mobile)_
-* Edge
 
-Others will probably work as well especially if Chromium based, but there might be slight differences.
+#### 3.4 Setup: RoBrowser
 
-__We assume in the guide below http server to run on port `8000`.__
+First, you will need the project in your machine. You can fork your own version or just clone the repository by typing:
 
-### Compile files using NPM
-This step/section is only recommended for a "Live" server. It will only pack all the resource files into one file to speed up loading. Requires to set in the roBrowser config: `development: false,`.
-- Access the roBrowserLegacy Folder using terminal or cmd `cd path/to/robrowserlegacy`
-- Use build command to generate the files `npm run build`
-- Files will be generated at `dist/Web` folder
-- Copy/move the generated files into your web folder (`Online.js` and `ThreadEventHandler.js`)
+```
+git clone https://github.com/MrAntares/roBrowserLegacy.git
+```
+
+Or you can just download our latest release [here.](ttps://github.com/MrAntares/roBrowserLegacy/archive/refs/heads/master.zip)
+
+Having the project extracted in your machine, you will need to install the node dependencies by running:
+
+```shell
+npm install
+```
+
+
+
+<!-- - Get a web server that supports `PHP` and `.htaccess`. Run http server at the root of the roBrowserLegacy directory, or place roBrowserLegacy into your webserver's content directory (use any of [one liner http server](https://gist.github.com/willurd/5720255))
+![](img/start-http-server.png)
+- You own a full client
+
+__We assume in the guide below http server to run on port `8000`.__ -->
 
-For development purposes (modifying the source/testing) set in the roBrowser config: `development: true,`.
-- Access the roBrowserLegacy Folder using terminal or cmd `cd path/to/robrowserlegacy`
-- Use build command to generate the files `npm run serve`
-- Files will be loaded from  at `src/` folder
+### 4. Compiling and Running roBrowser
+
+Now we have everything to make our game run as expected. Before start, we have to start a web-server to host all our roBrowserLegacy. 
+
+If you're using PHP or Ruby you can type any of these commands:
```

Comment:
PHP and Ruby are not listed in the prerequisites. Are they required prerequisites?

Reply:
Hmm any built-in server work tbh. Actually idk why not use only NodeJS for that since it's already installed. 

In my case I referenced because previously it was already there with this image and IDK how important it is for the project or if it's just to add some perspective to beginners. 

![image](https://github.com/MrAntares/roBrowserLegacy/assets/6912596/290cea0b-3260-4b62-a818-145dc1de1faa)

What do you guys think? @alisonrag @waken22

Reply:
The prerequisite is a web server, nothing more. You can use node for this too. The ruby was added by someone who changed the start.md, we can put there nodejs if it's preferable.

Reply:
when i wrote the start.md, the actual sentence was "Run http server at the root of the roBrowserLegacy directory, or place roBrowserLegacy into your webserver's content directory (use any of [one liner http server](https://gist.github.com/willurd/5720255))" there was no mention of technology, the screenshot just shown from which directory to serve content

Reply:
I decided to add a new command on **package.json** to open the live-server browserless but still let the mentions on other ways to open the server as well.